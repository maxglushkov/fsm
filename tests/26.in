2
2
S 0 A
A 1 S
