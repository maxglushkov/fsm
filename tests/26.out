2
2
S0S 0 S1M
S1M 1 S0S
